x-pack/plugins/lens/public/indexpattern_datasource/dimension_panel/format_selector.tsx

```diff
@@ -44,7 +55,8 @@ interface State {
   decimalPlaces: number;
 }
 
-const singleSelectionOption = { asPlainText: true };
+const RANGE_MIN = 0;
+const RANGE_MAX = 20;
```

Comment:
I think we can reduce this to 15?

JS Number cannot represent a number beyond the 17th digit: 
>  A Number only keeps about 17 decimal places of precision

(source: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number )

and 15 is the closest round number.